feedback/readme.md

```diff
@@ -0,0 +1,143 @@
+# Feedback Workshop 🗣️
+
+Feedback is the foundation of effective collaboration. Both giving and receiving feedback are important communication skills to master. 
```

Comment:
Excellent to see motivation here 🙏 

Let's rewrite this so it's accessible as possible to our B2 speakers. Hemingway can help here

https://hemingwayapp.com/

Aim for grade 4-6, no sentences hard or very hard to read.